src/AppInstallerCLITests/SQLiteIndex.cpp

```diff
@@ -35,49 +35,39 @@ SQLiteIndex CreateTestIndex(const std::string& filePath, std::optional<Schema::V
     // If no specific version requested, then use generator to run against the last 3 versions.
     if (!version)
     {
-        version = GENERATE(Schema::Version{ 1, 2 }, Schema::Version{ 1, 3 }, Schema::Version{ 1, 4 }, Schema::Version::Latest());
+        version = GENERATE(Schema::Version{ 1, 5 }, Schema::Version{ 1, 6 }, Schema::Version::Latest());

```

Comment:
It seem to me that because this is always for the last 3 versions, that there should be an operator method to be able to subtract versions from each other; something like `Schema::Version::Latest() - Schema::Version{0, 2}` so these don't need to be updated manually